README.md

```diff
@@ -18,6 +18,10 @@ We know it can be difficult to get into the whole open source thing at first. To
 Random halloween facts are posted regularly.  
 !hallofact - Show the last posted Halloween fact
 
+### Time Left
+Calculates the time remaining in Hacktoberfest. <br>
+.timeleft - Returns the days/hours/minutes/seconds left before Hacktoberfest ends.
```

Comment:
This could probably be clarified slightly, how is the output actually going to be formatted?